docs/auth0_api.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+---
+## auth0 api
+
+Makes an authenticated HTTP request to the Auth0 Management API
+
+### Synopsis
+
+Makes an authenticated HTTP request to the Auth0 Management API and prints the response as JSON.
+
+The method argument is optional, and when you don’t specify it, the command defaults to GET for requests without data and POST for requests with data.
+
+Auth0 Management API Docs:
+  https://auth0.com/docs/api/management/v2
+
+Available Methods:
+  GET, POST, PUT, PATCH, DELETE
+
+```
+auth0 api <method> <uri> [flags]
+```
+
+### Examples
+
+```
+auth0 api "/stats/daily?from=20221101&to=20221118"
+auth0 api get "/tenants/settings"
+auth0 api clients --data "{\"name\":\"ssoTest\",\"app_type\":\"sso_integration\"}"
+
+```
+
+### Options
+
+```
+  -d, --data string   JSON data payload to send with the request.
+  -h, --help          help for api
+```
+
+### Options inherited from parent commands
+
+```
+      --debug           Enable debug mode.
+      --force           Skip confirmation.
```

Comment:
Would it make sense to ask the user for confirmation when issuing a DELETE request, and leverage this flag to omit the prompt (similar to other `delete` commands, like https://auth0.github.io/auth0-cli/auth0_apps_delete.html)?

Otherwise, any way we can skip this flag for this command?

Reply:
Good suggestion, let me try to add it for the delete.

Reply:
Updated in: [`b16e634` (#531)](https://github.com/auth0/auth0-cli/pull/531/commits/b16e634b8466735eb652a2ba5bc1300d2b0d6bc4).